JustTrack/JEEventsGenerator/main.swift

```diff
@@ -348,7 +358,7 @@ do {
     //write struct file
     let structSwiftFilePath = CommandLine.arguments[2]
     if !FileManager.default.fileExists(atPath: structSwiftFilePath) {
-        throw JEStructsGeneratorError.swiftFileNotFound
+        FileManager.default.createFile(atPath: structSwiftFilePath, contents: .none, attributes: .none)
```

Comment:
Absolutely! @ssouzawallace thank you very much :) your help is much appreciated.
I've raised 2 issues, have a look: https://github.com/justeat/JustTrack/issues/5 and we can use this PR or implement it separately.

About the errors: I think that the best approach is to remove the crashes and improve the error logging at compiling time, making the script fail and printing the errors in the .swift file.